src/queries/makeQueries.ts

```diff
@@ -83,6 +85,39 @@ Example:
   return waitForOptions;
 }
 
+function getWrongHostComponentNamesError(): string {
```

Comment:
naming, nit: wrong -> incorrect:
```suggestion
function formatIncorrectHostComponentNamesError(): string {
```

Reply:
BTW in case the error is not relevant I would rather return `null` instead of empty string, as `null` feels like more natural representation of not having error than empty string.